.github/ISSUE_TEMPLATE/2.feature-request.yml

```diff
@@ -0,0 +1,37 @@
+name: 💡 Feature request
+description: Request a new feature, or improvements to existing features
+labels: ['status: unconfirmed']
+projects: ['folkehelseinstituttet/12']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to add a feature request!
+
+        > [!TIP]
+        > We accept content written in norwegian if that is your preferred language.
+        > _Vi aksepterer innhold skrevet på norsk hvis det er ditt foretrukne språk._
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe your proposed feature. Also, to give us some more context, describe the end user experience that led to the need for this feature.
```

Comment:
Describe the feature or improvement in as much detail possible, including the experience that led to its need and how you would use the new feature.